--- configure.orig	2021-01-01 12:41:16.000000000 +0200
+++ configure	2022-01-09 14:38:45.687986374 +0200
@@ -14549,42 +14561,6 @@
     CFLAGS="-Wall -Wno-enum-compare $CFLAGS"
 fi
 
-if test "$cross_compiling" = yes; then :
-  { { $as_echo "$as_me:${as_lineno-$LINENO}: error: in \`$ac_pwd':" >&5
-$as_echo "$as_me: error: in \`$ac_pwd':" >&2;}
-as_fn_error $? "cannot run test program while cross compiling
-See \`config.log' for more details" "$LINENO" 5; }
-else
-  cat confdefs.h - <<_ACEOF >conftest.$ac_ext
-/* end confdefs.h.  */
-
-#include <stdint.h>
-#include <string.h>
-
-int
-main ()
-{
-
-  volatile char *s = strdup("_abbaab_");
-  return *(volatile uint32_t*)(s + 1) != 0x61626261 ||
-         *(volatile uint32_t*)(s + 3) != 0x62616162;
-
-  ;
-  return 0;
-}
-_ACEOF
-if ac_fn_c_try_run "$LINENO"; then :
-
-
-$as_echo "#define CONFIG_ASSH_NONALIGNED_ACCESS /**/" >>confdefs.h
-
-
-fi
-rm -f core *.core core.conftest.* gmon.out bb.out conftest$ac_exeext \
-  conftest.$ac_objext conftest.beam conftest.$ac_ext
-fi
-
-
 ac_config_files="$ac_config_files libassh.pc Makefile src/Makefile src/assh/Makefile examples/Makefile test/Makefile doc/Makefile"
 
 cat >confcache <<\_ACEOF
